WORKSPACE

```diff
@@ -19,20 +19,25 @@ python_register_toolchains(
     python_version = "3.9",
 )
 
-load("@rules_python//python:pip.bzl", "pip_parse")
 load("@python39//:defs.bzl", "interpreter")
 
-pip_parse(
+load("@rules_python//python/pip_install:repositories.bzl", "pip_install_dependencies")
+
+pip_install_dependencies()
```

Comment:
pip_repository does not call this setup macro the way that pip_parse does, not sure if this is intentional, could be worth a bug report in rules_python, or a docs change to say that if you're planning to use any of the pip rules, you should call this.